### PR TITLE
Reviewed liblas quickstart

### PR DESCRIPTION
**Pull request by flicstar**
Reviewed as part of the QuickStart project during Google Season of Docs 2019. The aim is consistency across all OSGeoLive QuickStarts.

I added Intro and What next? sections. @hobu Please action comments and suggestions you agree with, and ignore those you don't. 

Partner trac ticket link: https://trac.osgeo.org/osgeolive/ticket/2211

See this page for reference: https://trac.osgeo.org/osgeolive/wiki/How%20to%20document%20the%20quickstart%20file

### CODE REVIEW

File: doc/quickstart/liblas_quickstart.rst [hunk 1]
```diff
@@ -24,7 +28,7 @@ you might want to do on LAS data, and the utilities and approaches to
 take to complete those tasks. The demo data can be found in :file:`/home/user/data/las/srs.las`
```
Comment (flicstar) [inline doc/quickstart/liblas_quickstart.rst]: Yes, this demo data srs.las file exists but I couldn't follow the procedure.

Comment (kalxas) [reply doc/quickstart/liblas_quickstart.rst]: Perhaps this is a file permissions issue.

**Comment by camerons** [comment]
As noted by Felicity (below) it looks like Liblas should be retired with the next 14.0 release.
I've reviewed this current quickstart updates and they look fine. We should keep this Quickstart description for 13.1 as the software is still included in the 13.0 release.

--

Felicity Brand via lists.osgeo.org 
5 Dec 2019, 10:02
to Osgeolive

During my work on the Quickstarts, a few things came up that I need to
raise more broadly. One of these was a comment Howard made about Liblas:

"The project is in dormant maintenance mode at this time. It has been
superseded by the PDAL project."

I'm sharing this to start a conversation.

Thanks
Felicity

**Comment by kalxas** [comment]
The project has been dropped from Debian, so it will drop from Ubuntu soon as well.

**Pull request merged**